vault/resource_consul_secret_backend_test.go

```diff
@@ -24,83 +29,128 @@ func TestConsulSecretBackend(t *testing.T) {
 			{
 				Config: testConsulSecretBackend_initialConfig(path, token),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "path", path),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "description", "test description"),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "default_lease_ttl_seconds", "3600"),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "max_lease_ttl_seconds", "86400"),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "address", "127.0.0.1:8500"),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "token", token),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "scheme", "http"),
-					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "local", "false"),
-					resource.TestCheckNoResourceAttr("vault_consul_secret_backend.test", "ca_cert"),
-					resource.TestCheckNoResourceAttr("vault_consul_secret_backend.test", "client_cert"),
-					resource.TestCheckNoResourceAttr("vault_consul_secret_backend.test", "client_key"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, path),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDescription, "test description"),
+					resource.TestCheckResourceAttr(resourceName, "default_lease_ttl_seconds", "3600"),
+					resource.TestCheckResourceAttr(resourceName, "max_lease_ttl_seconds", "86400"),
+					resource.TestCheckResourceAttr(resourceName, "address", "127.0.0.1:8500"),
+					resource.TestCheckResourceAttr(resourceName, "token", token),
+					resource.TestCheckResourceAttr(resourceName, "scheme", "http"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldLocal, "false"),
+					resource.TestCheckNoResourceAttr(resourceName, "ca_cert"),
+					resource.TestCheckNoResourceAttr(resourceName, "client_cert"),
+					resource.TestCheckNoResourceAttr(resourceName, "client_key"),
```

Comment:
Probably want to add some import test steps in this test, since we dropped the dedicated import test case.